module/zfs/dsl_scan.c

```diff
@@ -129,6 +129,7 @@ static void scan_ds_queue_sync(dsl_scan_t *scn, dmu_tx_t *tx);
 static uint64_t dsl_scan_count_data_disks(vdev_t *vd);
 
 extern int zfs_vdev_async_write_active_min_dirty_percent;
+static int zfs_scan_blkstats;
```

Comment:
```suggestion
static int zfs_scan_blkstats = 0;
```

Reply:
On FreeBSD it is not recommended, but whatever.  Done.